ui/components.d.ts

```diff
@@ -58,8 +52,6 @@ declare module 'vue' {
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
-    NUploadFileList: typeof import('naive-ui')['NUploadFileList']
-    NUploadTrigger: typeof import('naive-ui')['NUploadTrigger']
     NWatermark: typeof import('naive-ui')['NWatermark']
     PamFileList: typeof import('./src/components/pamFileList/index.vue')['default']
     RouterLink: typeof import('vue-router')['RouterLink']
```

Comment:
There is no major code difference to address, but few suggestions:

1. Remove unused modules (`@@`, `+`) that were originally present.
2. Reorganize some imports (if necessary).
3. Optimize import statements. For example: use ES6 aliasing where appropriate.